amy/autoemails/actions.py

```diff
@@ -1133,3 +1134,66 @@ def get_additional_context(self, objects, *args, **kwargs):
             )
 
         return context
+
+
+class NewConsentRequiredAction(BaseAction):
+    """
+    Action for asking users to consent to newly required terms. This email
+    should be sent when a new required Term is created.
+
+    How to use it:
+
+    >>> triggers = Trigger.objects.filter(active=True,
+                                          action='consent-required-new')
+    >>> for trigger in triggers:
+    ...     action = NewConsentRequiredAction(
+    ...         trigger=trigger,
+    ...         objects=dict(terms=terms),
+    ...     )
+    ...     launch_at = action.get_launch_at()
+    ...     job = scheduler.enqueue_in(launch_at, action)
+    """
+
+    launch_at = timedelta(hours=1)
+
+    def get_launch_at(self):
+        return self.launch_at
+
+    def recipients(self) -> Optional[str]:
+        """Assuming self.context is ready, overwrite email's recipients
+        with selected ones."""
+        try:
+            return self.context["all_emails"]
+        except (AttributeError, KeyError):
+            return None
+
+    def all_recipients(self) -> str:
+        """If available, return string of all recipients."""
+        # TODO what is this versus recipients?
```

Comment:
One is used in sending the action, the other is used for displaying in Django Admin panel. I don't remember which one is which.